assets/src/edit-story/app/api/getAllPageLayouts.js

```diff
@@ -67,12 +67,16 @@ const replacePageImagesWithPlaceholders = (page, { assetsURL }) => {
   };
 };
 
-export default async function getAllPageLayouts({ cdnURL, assetsURL }) {
+export default async function getAllPageLayouts({
+  cdnURL,
+  assetsURL,
+  showImages = true,
```

Comment:
Looks like the default here is `true` for `showImages` but above in `getPageLayouts` it's `false` -- is this purposefully so? Should the default always be `false` as that's the default in Figma? Thoughts?